content/getting-started/index.md

```diff
@@ -1,343 +1,37 @@
 ---
-title: Programming the Clawbot
+title: Getting Started with PROS
 ---
 
-#### Objective:
-This tutorial will guide you through basic programming of the VEX Clawbot.
+## Should I use PROS?
+Since you're here and reading our documentation, the answer is most likely yes! However, PROS (and C programming in general) can be difficult for a lot of beginners, so it may not be the ideal solution for everyone.
 
-#### Intended Audience:
-This tutorial is intended for developers with some programming experience, but with little to no experience with the PROS library. If you haven't programmed before, we recommend checking out all the "Introduction and Basic C Features" sections of [this tutorial series](http://www.cprogramming.com/tutorial/c-tutorial.html); you may also benefit from the "Pointers, Arrays and Strings" sections as well (although they aren't as pertintent).
+If you find yourself saying the following things often:
 
-#### Goals:
-At the end of this tutorial you will have:
+* "I don't want to actually learn to code, I just want some code to make my awesome mechanical design move!"
+* "I'm not sure I like coding, but that PROS Editor's dark theme though..."
+* "Reading documentation is for losers, I'm sure I can get my code working with trial and error!"

```

Comment:
I feel like there's a better way to phrase this.

Reply:
Are you looking for more legitimate sounding reasons, or simply a reworking of the phrasing? I felt that it might be best to keep the reasons for not using PROS somewhat light-hearted and ridiculous to avoid sounding like we're trying to push people away, but I completely understand that such an approach can easily come across the wrong way.

Do you want to keep those three ideas but rewrite them, or go for different ideas entirely?

Reply:
I didn't notice this before, but now that I'm reading this again, I have to agree with Elliot here. One stand-out example is the "reading documentation is for losers" comment. It's a little too standoffish. In general, let's avoid disparaging (or appearing to disparage) people who don't end up using PROS.

Reply:
How about something like this:

* I really just want a quick, minimal software solution to make my robot move.
* I don't want to have to look through documentation to write code, the programming environment should prevent errors from occurring.

or something along those lines, I can't really think of a good way to write the last point.

Reply:
I'm okay with the first one. Also tentatively okay with the second, although it still feels suspiciously like a needle towards _a certain programming environment_. What does @edjubuh think?

Reply:
Yeah I'm really leery of hinting too heavily towards other programming environments here. I thought about discussing debugging (printf versus a debugger), but figured that also might be too pointed.